### DISCUSSION TITLE
FOCUS GROUP ON AUTONOMOUS NETWORKS (FG-AN)

### DISCUSSION THREAD

**Discussion by rajat-k8** [vrra/FGAN-Build-a-thon-2022#27] [Q&A (General)] [answered]
How we will get the data set for use cases?

**Comment by CrashingGuru** [comment, marked as answer]
Hi,

Thank you for the question @rajat-k8 

Short answer: 
You can execute the notebook 
https://github.com/vrra/FGAN-Build-a-thon-2022/blob/main/Notebooks2022/build_a_thon_graph_v1.ipynb
and it will generate a database full of data 

Long answer: https://github.com/vrra/FGAN-Build-a-thon-2022/discussions/10

Any follow up questions are welcome!
